test/integration/proportion-boolean/patients/patient-denom-bundle.json

```diff
@@ -0,0 +1,178 @@
+            }
+          ]
+        },
+        "status": "completed",
+        "subject": {
+          "reference": "Patient/22af5434-67a9-4f51-8a74-bf3f2d62141d"
+        },
+        "performedPeriod": {
+          "start": "2022-07-25T02:43:18.343Z",
+          "end": "2022-07-26T02:43:18.343Z"
+        }
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Procedure/7d3793bb-6ba4-423d-a642-73e51566e8cc"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Procedure",
+        "id": "4faf425c-983b-4518-9a97-754b5d791638",
+        "code": {
+          "coding": [
+            {
+              "system": "exampleSystem2",
+              "version": "exampleVersion2",
+              "code": "exampleCode2"
+            }
+          ]
+        },
+        "status": "completed",
+        "subject": {
+          "reference": "Patient/22af5434-67a9-4f51-8a74-bf3f2d62141d"
+        },
+        "performedPeriod": {
+          "start": "2022-06-27T01:29:58.417Z",
+          "end": "2022-06-28T01:29:58.417Z"
+        }
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Procedure/4faf425c-983b-4518-9a97-754b5d791638"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "MeasureReport",
+        "id": "6807e7f4-bec8-4122-aec2-0f0a1be16bca",
+        "measure": "http://example.com/Measure/measure-ProportionBooleanTest",
+        "period": {
+          "start": "2022-01-01T05:00:00.000Z",
+          "end": "2022-12-31T05:00:00.000Z"
+        },
+        "status": "complete",
+        "type": "individual",
+        "meta": {
+          "profile": [
+            "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/test-case-cqfm"
+          ]
+        },
+        "extension": [
+          {
+            "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-inputParameters",
+            "valueReference": {
+              "reference": "#4522e13a-f764-44de-82c5-409e6176e644"
+            }
+          }
+        ],
+        "modifierExtension": [
+          {
+            "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-isTestCase",
+            "valueBoolean": true
+          }
+        ],
+        "contained": [
+          {
+            "resourceType": "Parameters",
+            "id": "4522e13a-f764-44de-82c5-409e6176e644",
+            "parameter": [
+              {
+                "name": "subject",
+                "valueString": "22af5434-67a9-4f51-8a74-bf3f2d62141d"
+              }
+            ]
+          }
+        ],
+        "group": [
+          {
+            "population": [
+              {
+                "code": {
+                  "coding": [
+                    {
+                      "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+                      "code": "initial-population"
+                    }
+                  ]
+                },
+                "count": 0
```

Comment:
It doesn't _really_ matter for the sake of this integration test, but maybe in the future if we ever want to do anything with the test case measure report, we should properly reflect it in fqm-testify.

AKA can you actually select the desired populations in testify before exporting the patient?

Reply:
Gotcha. We were on the fence about this and opted not to. Happy to add them back :)